app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -203,18 +219,37 @@ def kill_middleware_domain_server(ems_ref)
       run_generic_operation(:Kill, ems_ref)
     end
 
-    def shutdown_middleware_server(ems_ref, params)
+    # server group ops
+    def start_middleware_server_group(ems_ref)
+      # blocking: boolean
+      run_generic_operation('Start Servers', ems_ref)
+    end
+
+    def stop_middleware_server_group(ems_ref, params)
+      # blocking: boolean
+      # timeout: int
```

Comment:
So is `params` a hash that is expected to have `{:blocking, :timeout}` as possible keys?